docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -25,91 +26,254 @@ Expo as a framework enables **Continuous Native Generation** by combining the fo
+
+</Collapsible>
+
+<Collapsible summary="Is CNG limited to React Native projects?">
 
 No, CNG is a versatile pattern that can be applied to any native project. While Expo Prebuild is a tool that implements CNG specifically for React Native projects, the concept itself is not limited to this framework.
 
-### Can CNG be used for operating systems other than Android and iOS?
+</Collapsible>
+
+<Collapsible summary="How does community uses CNG?">
+
+Here are a few community examples of difficult native features converted into simple configuration files, which has allowed developers to build more powerful apps without compromising on iteration speed:
+
+- [iOS Safari Extensions](https://github.com/andrew-levy/react-native-safari-extension): You can see the process of creating a Safari Extension for iOS, which is a notoriously difficult feature to implement, boiled down to a couple of lines of JSON.
+
+- [iMessage Sticker App](https://github.com/expo/config-plugins/tree/main/packages/ios-stickers): This Expo Config Plugin can generate an entire iMessage Sticker App from a JSON object.
+
+- [Cross-platform End-2-End testing](https://github.com/expo/config-plugins/tree/main/packages/detox): Configure native apps to support E2E testing with Detox in a single-line.
+
+- [Entire Firebase suite](https://rnfirebase.io/): Here you can see the entire native Firebase suite going from a multi-step native process across multiple IDEs, down to basic JSON configuration.
 
-Absolutely! CNG is an abstract concept that can be applied to any operating system. Take Expo for example, although Expo Prebuild officially implements CNG for Android and iOS, it also provides abstract platform support for users to create implementations for additional platforms.
+- [Cross-platform Home screen Widgets](https://github.com/gaishimo/eas-widget-example): This Expo Config Plugin can generate a home screen widget for Android and iOS.
 
-### Is using Expo a requirement for CNG?
+- [Notification extension and code signing](https://github.com/OneSignal/onesignal-expo-plugin): This Expo Config Plugin not only generates a notification extension target on iOS. However, it also augments the EAS credentials service to keep zero-config code signing working.
+
+- [Apple App Clips](https://github.com/bndkt/react-native-app-clip): This Expo Config Plugin takes the process of generating an Apple App Clip from a multi-step process, ranging across multiple targets, and shakes it down to a single line `["react-native-app-clip", { "name": "My App Clip" }]`.
+
+At any point, these features can be easily added and removed, without any side effects. CNG allows developers to experiment with complex features, and iterate on them quickly without worrying about the long-term maintenance costs or potential orphaned code in their project or uncommitted native caches.
```

Comment:
```suggestion
At any point, these features can be easily added and removed, without any side effects. CNG allows developers to experiment with complex features and iterate on them quickly without worrying about the long-term maintenance costs or potential orphaned code in their project, or uncommitted native caches.
```
Some of this sound fluffy... not quite sure what an "uncommitted native cache" is.

Reply:
I've decided to remove this as I am not sure what this means.